### PR TITLE
Impossible to use destination rate groups file importer arguments from api

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [x] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
This issue inserts new data for destinationRateGroups petitions.

### CODE REVIEW

File: library/Ivoz/Provider/Domain/Model/DestinationRateGroup/FileImporterArguments.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    private function getIgnoreFirst(): bool
+    {
+        return $this->ignoreFirst;
+    }
+
+    public function setIgnoreFirst(bool $ignoreFirst): void
+    {
+            $this->ignoreFirst = $ignoreFirst;
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/DestinationRateGroup/FileImporterArguments.php:82]: fix indentation here

**Review by mmadariaga — CHANGES_REQUESTED** [review]
Update API tests as well

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/rest/brand/features/provider/destinationRateGroup/postDestinationRateGroup.feature [hunk 2]
```diff
@@ -64,7 +64,19 @@ Content-Type: text/csv
               "fileSize": 84,
               "mimeType": "text/plain; charset=us-ascii",
               "baseName": "prices.csv",
-              "importerArguments": null
+              "importerArguments": {
```
Comment (mmadariaga) [inline web/rest/brand/features/provider/destinationRateGroup/postDestinationRateGroup.feature:67]: Missing importer arguments in POST

File: web/rest/brand/features/provider/destinationRateGroup/putDestinationRateGroup.feature [hunk 3]
```diff
@@ -64,3 +64,82 @@ Feature: Update destination rate group
...
+        "ca": "New DR",
+        "it": "New DR"
+    },
+    "description": {
+        "en": "",
+        "es": "",
+        "ca": "",
+        "it": ""
+    },
+    "currency": "1"
```
Comment (mmadariaga) [inline web/rest/brand/features/provider/destinationRateGroup/putDestinationRateGroup.feature:91]: Missing importer arguments

File: web/rest/brand/features/provider/destinationRateGroup/putDestinationRateGroup.feature [hunk 4]
```diff
@@ -64,3 +64,82 @@ Feature: Update destination rate group
           "currency": 2
       }
     """
+
+  @createSchema
+  Scenario: Update a destination rate group
```
Comment (mmadariaga) [inline web/rest/brand/features/provider/destinationRateGroup/putDestinationRateGroup.feature:69]: This test replaces the former one

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: library/Ivoz/Provider/Domain/Model/DestinationRateGroup/DestinationRateGroupDto.php [hunk 5]
```diff
@@ -43,10 +72,44 @@ public function denormalize(array $data, string $context, string $role = ''): vo
             $contextProperties['file'][] = 'path';
         }
 
+        $importerArguments = null;
+        $includeImporterArgumens = !in_array($context, self::CONTEXTS_WITHOUT_IMPORTER_ARGUMENTS);
```
Comment (ironBatista) [inline library/Ivoz/Provider/Domain/Model/DestinationRateGroup/DestinationRateGroupDto.php:76]: Possible typo? $includeImporterArgumens -> $includeImporterArguments

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**